java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S6355_java.html

```diff
@@ -0,0 +1 @@
+<p>FIXME, see https://github.com/SonarSource/rspec/pull/290</p>
```

Comment:
When the review of this PR will be done, we should review and merge the other PR first (https://github.com/SonarSource/rspec/pull/290), and then update the metadata.